spec/support/stories/profile_stories.rb

```diff
@@ -9,6 +9,8 @@ def as_a_registered_user
       email: email, password: password, confirmed_at: confirmed_at,
       account: Fabricate(:account, username: 'bob')
     )
+
+    Web::Setting.where(user: bob).first_or_initialize(user: bob).update!(data: { introductionVersion: 201812160442020 }) if finished_onboarding # rubocop:disable Style/NumericLiterals
```

Comment:
Is this related to the PR?

Reply:
Yes, I needed a way to create users who do not have the onboarding screen for the 2 specs I wrote